EKF/ekf_helper.cpp

```diff
@@ -708,6 +708,31 @@ void Ekf::getFlowInnovRatio(float &flow_innov_ratio) const
 	flow_innov_ratio = _optflow_test_ratio;
 }
 
+Vector2f Ekf::getFlowVelBody() const
+{
+	return _flow_vel_body;
```

Comment:
Simple one liners like this can be kept in the header to allow for potential inlining across translation units (ie ekf2 calling ecl/EKF).

Reply:
Yes, will do that next time